atomic_defi_design/Dex/Components/GasInfoTrigger.qml

```diff
@@ -0,0 +1,8 @@
+import QtQuick 2.15
```

Comment:
No need to create a new component it's used only in one place
Moreover using directly `gas_info_modal` here leads to undefined behavior since you are not 100% sure this component will be used inside the `Dashboard` page which contains the `gas_info_modal` definition

Reply:
I've created `DefaultInfoTrigger` component and replaced using `CexInfoTriger` and `GasInfoTrigger` with this.